scripts/contributor-workflow-setup.sh

```diff
@@ -66,7 +66,10 @@ mkdir --parents tmp/root
 banner "MAINTAINER"
 ###################
 
-rad auth --init --name cloudhead --passphrase cloudhead
+export RAD_PASSPHRASE=cloudhead
+rad auth --init --name cloudhead
+# Alternatively, the passphrase can be passed via stdin:
+# cloudhead | rad auth --init --name cloudhead --stdin
```

Comment:
Perhaps that's cleaner for the script? I think it would be `echo cloudhead | rad auth ...` - not sure if you need the `-n` flag as well..

Reply:
Just checked, and `-n` is not needed.